.gitattributes

```diff
@@ -23,6 +23,7 @@
 /package.json export-ignore
 /package-lock.json export-ignore
 /phpcs.xml.dist export-ignore
+/phpcs.ruleset.xml export-ignore
 /phpstan.neon.dist export-ignore
 /phpunit.xml.dist export-ignore
```

Comment:
I think we'll also have to cater here for the individual `/plugins/**/phpcs.xml.dist` files now that they may exist per individual plugin.

If I'm not mistaken, without this the standalone plugin builds would include these files, but they should be omitted.

Reply:
Right, although I think actually we should rather do this:

```diff
- /phpcs.xml.dist export-ignore
+ phpcs.xml.dist export-ignore
```

We should just ignore these PHPCS files anywhere in the repo from being exported.